src/main/java/crazypants/enderio/machine/spawner/ItemBrokenSpawner.java

```diff
@@ -35,14 +35,20 @@ public static String getMobTypeFromStack(ItemStack stack) {
     }
     return stack.stackTagCompound.getString("mobType");
   }
-  
-  public static ItemStack createStackForMobType(String mobType) {
+
+  public static ItemStack createStackForMobType(String mobType)
+  {
+    return createStackForMobType(mobType,false);
+  }
+
+  public static ItemStack createStackForMobType(String mobType, boolean wither) {
```

Comment:
Wither Skeletons are the only entities that behave like this - in Minecraft itself or in fact any mod that I know of. If you look at other mods that have to differentiate like this, eg MopProperties: http://puu.sh/e73rn/b705442d17.png they all do the same